lib/postgresql.js

```diff
@@ -15,6 +15,11 @@ var ParameterizedSQL = SqlConnector.ParameterizedSQL;
 var util = require('util');
 var debug = require('debug')('loopback:connector:postgresql');
 
+// Add support for node 0.10.x
+if (typeof Promise == 'undefined') {
+  global.Promise = require('promise-polyfill');
```

Comment:
We should be using bluebird for v0.10, Promise is not defined because native promises (global.Promise) are not supported in v0.10.